modules/boulder/CHANGELOG.md

```diff
@@ -0,0 +1,12 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## Unreleased
```

Comment:
We maybe should have a `Changed` here to state that we replaced ADTs?

Reply:
Did this, unsure about the "standard functions" wording though. Thoughts?